lockbox-ios/TelemetryIntegration.swift

```diff
@@ -9,14 +9,29 @@ class TelemetryEventCategory {
 }
 
 class TelemetryEventMethod {
+    public static let startup = "startup"
     public static let background = "background"
```

Comment:
I don't see a "backgrounded" or "foregrounded" event in the above enumeration, so these can be removed as well.